plugin-server/tests/worker/ingestion/person-state.test.ts

```diff
@@ -455,7 +455,7 @@ describe('PersonState.update()', () => {
         const distinctIds = await hub.db.fetchDistinctIdValues(persons[0])
         expect(distinctIds).toEqual(expect.arrayContaining(['new-user', 'old-user']))
 
-        const clickhousePersons = await delayUntilEventIngested(() => fetchPersonsRows({ final: true }), 2)
+        const clickhousePersons = await delayUntilEventIngested(() => fetchPersonsRows(), 2)
```

Comment:
Context? This was added indentionally.

Reply:
This function doesn't take any params.

Not on this file at least - the one on another file does